docs/superpowers/plans/2026-05-15-tracker-git-preflight-plan.md

```
@@ -0,0 +1,2927 @@
+grep -lE 'git (commit|add|push|tag|status|init|checkout|branch|stash|merge|rebase)' workflows/*.dip
+```
+
+Expected output indicates which workflows reference git commands in prompts or tool blocks. Compare against the spec's list. The spec calls out `build_product`, `build_product_with_superspec`, and (in tracker's installed name) potentially others.
+
+Note any workflow that surfaces git but whose authors didn't intend a hard requirement — those get the comma-separated form anyway since `requires: git` is the gate.
+
+- [ ] **Step 2: Add `requires: git` to build_product.dip**
+
+Edit `workflows/build_product.dip` lines 1-4:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)